controllers/discordactions.js

```diff
@@ -121,7 +120,41 @@ const addGroupRoleToMember = async (req, res) => {
     return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
   }
 };
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeNicknameOfUsers = async (req, res) => {
+  try {
+    const userData = req.userData;
 
+    const { discordId, username } = userData;
+
+    const discordData = {
+      userName: username,
+      discordId,
+    };
+
+    const authToken = await jwt.sign({}, config.get("rdsServerlessBot.rdsServerLessPrivateKey"), {
+      algorithm: "RS256",
+      expiresIn: config.get("rdsServerlessBot.ttl"),
+    });
+
+    await fetch(`${DISCORD_BASE_URL}/guild/member`, {
```

Comment:
we shouldn't use await and then together. Rather it is encouraged to use await over then for more predictive execution. Let me know if you need a bit more clarification.